tests/embedchain/test_add.py

```diff
@@ -0,0 +1,26 @@
+# ruff: noqa: E501
```

Comment:
Let's remove this from all the test files. We only want to specific specific lines and not all of the file.

Reply:
that's going to be a lot of lines and decrease readability.